openshift/clowder/clowd-app.yaml

```diff
@@ -1,82 +1,88 @@
-apiVersion: cloud.redhat.com/v1alpha1
-kind: ClowdApp
+apiVersion: v1
+kind: Template
 metadata:
   name: automation-hub
-  namespace: automation-hub
-spec:
-  # The name of the ClowdEnvironment providing the services
-  envName: automation-hub-env
```

Comment:
you may want to use `${ENV_NAME}` for the value and add a parameter `ENV_NAME` which will be assigned by e.g. the prod env value here https://gitlab.cee.redhat.com/service/app-interface/blob/master/data/products/insights/environments/production.yml#L53